Cargo.toml

```diff
@@ -30,13 +30,12 @@ unexpected_cfgs = { level = "warn", check-cfg = ["cfg(iroh_docsrs)"] }
 unused-async = "warn"
 
 [dependencies]
+iroh-metrics-derive = { path = "./iroh-metrics-derive", version = "0.1.0" }
+itoa = "1"
```

Comment:
This is for faster int to string encoding (prometheus_client used it for that purpose, thought it wouldn't hurt here either)